steamclog/src/main/java/com/steamclock/steamclog/Destinations.kt

```diff
@@ -48,12 +50,29 @@ internal class CrashlyticsDestination : Timber.Tree() {
     }
 
     override fun log(priority: Int, tag: String?, message: String, throwable: Throwable?) {
+        val wrapper = throwable as? SteamclogThrowableWrapper
```

Comment:
Are there any cases where we're expecting `throwable` to not be a `SteamclogThrowableWrapper`?

Reply:
With this PR we should only be getting `SteamclogThrowableWrapper`

Reply:
Should we use `as` instead of `as?` in this case then?

Reply:
Yeah, I was just thinking about this a little more. Maybe this should be slightly more flexible - I would hate for a crash to happen somewhere because in the app someone uses `Timber` instead of `sclog` by accident.

Reply:
`SteamclogThrowableWrapper.from` now enforces that we will always have a valid object moving forward.